pagopa/io-app
# chore(EU Covid Certificate): [IAGP-65] Add top and bottom qrcode spacing

## Short description
This pr adds some spacing between the textual elements and the QRCode.

Before            |  After
:-------------------------:|:-------------------------:
<img src="https://user-images.githubusercontent.com/26501317/122798399-d7bd6700-d2c0-11eb-9add-3b74eca59799.png" width="300">|  <img src="https://user-images.githubusercontent.com/26501317/122798483-f0c61800-d2c0-11eb-926d-805e4b9c1a99.png" width="300">

## List of changes proposed in this pull request
- Wrap the QRCode component with two `<View spacer={true} />` 

## How to test
Open a message with the EU Covid Certificate > check the top and bottom spacing.
